### PR TITLE
Dala 1285 add lei to UI

### PR DESCRIPTION
**Pull request by stewal19**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server "dev1" using this branch
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Execute `sudo apt-get update && sudo apt-get upgrade` on
   - [ ] It's verified that real data has been used
   - [ ] It's verified that everything seems to be working fine by manually using the website
 - [ ] This template has been updated to reflect the latest state of tasks required and known issues with upgrades
+- [ ] At least one commit needs to be called 'manual-maintenance' to satisfy the CI maintenance check in future commits
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:83]: That won't help - the MR commit needs to have "manual maintenace" in the description

File: dataland-frontend/tests/e2e/utils/CompanyUpload.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ export function fillCompanyUploadFields(companyName: string): void {
   cy.get("input[name=headquartersPostalCode]").type("123456", { force: true });
   cy.get("input[name=companyLegalForm]").type("Enterprise Ltd.", { force: true });
   cy.get("input[name=website]").type("www.company.com", { force: true });
+  cy.get("input[name=lei]").type(`LeiValueId:${crypto.randomUUID()}`, { force: true });
```
Comment (ppuetsch) [inline dataland-frontend/tests/e2e/utils/CompanyUpload.ts:28]: seemingly this does not work: according to the sonar coverage analysis, all identifiers are empty:

image


I would expect if we get this running, then the sonar coverage isn't an issue anymore.

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 1 above)
Comment (janherre) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md]: Adapted the reminder.

Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md]: ok

File: dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts [hunk 3]
```diff
@@ -208,8 +209,9 @@ describe("The shared header of the framework pages should act as expected", { sc
         cy.ensureLoggedIn();
         const someInvalidCompanyId = "12345-some-invalid-companyId";
         const someInvalidDataId = "789-some-invalid-dataId-987";
+        cy.intercept(`**/companies/${someInvalidCompanyId}/frameworks/${DataTypeEnum.Lksg}`).as("searchCompany");
         cy.visit(`/companies/${someInvalidCompanyId}/frameworks/${DataTypeEnum.Lksg}`);
-        cy.wait(10000);
+        cy.wait("@searchCompany", { timeout: Cypress.env("medium_timeout_in_ms") as number });
```
Comment (ppuetsch) [inline dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts:214]: Ich glaube das muss nochmal angepasst werden insgesamt - mit den Waits - das läuft aktuell nicht sehr stabil (siehe https://github.com/d-fine/Dataland/actions/runs/4414704137)

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 4]
```diff
@@ -80,6 +80,7 @@ Execute `sudo apt-get update && sudo apt-get upgrade` on
   - [ ] It's verified that real data has been used
   - [ ] It's verified that everything seems to be working fine by manually using the website
 - [ ] This template has been updated to reflect the latest state of tasks required and known issues with upgrades
+- [ ] The Merge Request commit message needs to contain 'manual maintenance' to satisfy the CI maintenance check in future commits
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:83]: @ahoecherl please add the part for RabbitMQ Dead Letter queue

Comment (ahoecherl) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:87]: Done

File: dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts (hunk 3 above)
Comment (stewal19) [reply dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts]: Wir haben das nochmal angepasst.

File: dataland-frontend/tests/e2e/utils/CompanyUpload.ts (hunk 2 above)
Comment (ppuetsch) [reply dataland-frontend/tests/e2e/utils/CompanyUpload.ts:28]: ok

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 4 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:87]: ok

File: dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts (hunk 3 above)
Comment (ppuetsch) [reply dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts]: ok

**Pull request merged**